### ISSUE TITLE
Fix CI integration

### ISSUE THREAD

**Issue by mr-c** [xenon-middleware/xenon-flow#20]
```
bash jenkins.bash

FAILURE: Build failed with an exception.

* What went wrong:
Could not resolve all dependencies for configuration ':runtime'.
> Could not find nl.esciencecenter.xenon:xenon:2.0.0.
  Required by:
      :xenon-cwl-runner:unspecified

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output.

BUILD FAILED
```

From https://ci.commonwl.org/job/xenon-cwl-runner-conformance/90/console

Thanks,

**Comment by bpmweel** [comment]
Hey @mr-c 

This repo is currently very volatile. It's built on a as of yet unreleased version of Xenon and needs a lot more polish before we're close to a release.

Perhaps we can disable the jenkins tests until we're closer to a release, by then we should also have actual tests for instance :smile: 

Thanks,

**Comment by bpmweel** [comment]
Actually the conformance tests require a command line tool, so this issue depends on #3

**Comment by bpmweel** [comment]
@mr-c I have made a github action that runs the conformance tests on xenonflow through a small python script as a runner: https://github.com/xenon-middleware/xenon-flow/actions/workflows/cwl_compliance.yml

I have added the badges generated by this to the README.md of this repository.

I have removed the jenkins file as it seems this is no longer used. I hope this is enough of an indication for xenonflows conformance.
